tests/unit/test_groupby.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from scmdata.testing import assert_scmdf_almost_equal
+
+
+@pytest.mark.parametrize(
+    "g", (("variable",), ("variable", "scenario"), ("variable", "region"), ("model",),)
+)
+def test_groupby(test_scm_run, g):
```

Comment:
can you add a quick explanation of what this is doing? Just making sure you get the same thing back?